frontend/src/pages/PropertyDetails.jsx

```
@@ -429,6 +412,182 @@ const PropertyDetails = () => {
+      
+        return (
+          <li className="flex items-center justify-between py-2 px-4 bg-gray-100 rounded-md shadow-sm">
+            <span className="text-gray-900 font-medium">
+              {formatDate(date.checkIn)} — {formatDate(date.checkOut)}
+            </span>
+            {isBooker && (
+              <button
+                onClick={() => onUnbook(date._id)}
+                className="bg-red-500 text-white text-xs font-semibold px-3 py-1 rounded-md hover:bg-red-600 transition duration-200"
+              >
+                Unbook
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)